header-gossip-network-test-vectors.md

```diff
@@ -0,0 +1,38 @@
+# Header Gossip Network Test Vectors
+
+This document contains test vectors for the Header Gossip Network to help implementations conform to the specification.
+
+## Header Accumulator Encodings
+
+These test vectors specify the proper encoding of the header accumulator in certain well defined scenarios
+
+### Epoch 0 Accumulator
+
+#### Accumulator after genesis block
+
+This test vector represents the input parameters and the expected output for the current epoch accumulator after adding
+the genesis block from Ethereum Mainnet.
+
+##### Input Parameters
+```
+genesis block hash = 0x88e96d4537bea4d9c05d12549907b32561d3bf31f45aae734cdc119f13406cb6
+genesis block difficulty = 17171480576
+```
+##### Expected Output
+```
+accumulator hash root = 0x1978df242d723405f28d26184e57ccf1938b253ef08234bfeb4951abdf3bbe4c
```

Comment:
Is this the `hash_tree_root()` from the resulting `Accumulator` ? (see also https://github.com/ethereum/consensus-specs/blob/1235e58a8db09efd058f31087d53d91b97b83011/ssz/simple-serialize.md#merkleization).

I'm getting different results on my first go at this.

Reply:
OK, I noticed it is the `hash_tree_root` of just the `EpochAccumulator`. If I print those, they match.
Perhaps change it to the value of hash_tree_root of the full `Accumulator`?

Reply:
I wasn't sure how to go on that so went with the current epoch accumulator as the historical hashes array was empty. Let me update my local tests and will report back.

Reply:
I switched from epoch accumulator to full header accumulator hash roots